sdk/communication/communication-sms/README.md

```diff
@@ -109,57 +109,69 @@ for (const sendResult of sendResults) {
 ```
 
 ## Check if a list of recipients is in the Opt Out list
+
 To check if the recipients are in the Opt Out list, call the `check` function from the `SmsClient.optOuts` with a list of recipient phone numbers.
 
-```typescript
-  const optOutCheckResults = await client.optOuts.check(
-    "<from-phone-number>", // Your E.164 formatted phone number used to send SMS
-    ["<to-phone-number-1>", "<to-phone-number-2>"], // E.164 formatted recipient phone numbers
-  );
+```ts snippet:ReadmeSampleOptOutCheck
+const optOutCheckResults = await client.optOuts.check(
+  "<from-phone-number>", // Your E.164 formatted phone number used to send SMS
+  ["<to-phone-number-1>", "<to-phone-number-2>"], // E.164 formatted recipient phone numbers
+);
 
-  for (const optOutCheckResult of optOutCheckResults) {
-    if (optOutCheckResult.httpStatusCode === 200) {
-      console.log("Success: ", optOutCheckResult);
-    } else {
-      console.error("Something went wrong when trying to send opt out check request: ", optOutCheckResult);
-    }
+for (const optOutCheckResult of optOutCheckResults) {
+  if (optOutCheckResult.httpStatusCode === 200) {
```

Comment:
Let's use isUnexpected?

Reply:
same in other places in this file

Reply:
Unfortunately since the project isn't an RLC, it doesn't have `isUnexpected`, and instead you have to handle the sucess and failure yourself.  Perhaps an addition to the next version of the package?